logger.py

```diff
@@ -110,12 +128,17 @@ def restore_logger_object(self, log):
 
         """Restore dict"""
         log = listify.delistify(log)
-        
+
+        self.network_data["unweighted_network_data"] = log["unweighted_network_data"]
```

Comment:
One more thing: I think in order for the next four lines to work with ensembles, these keys have to be added to the `requested_logs` dict in `ensemble.py`, line 52 and following.  This is because ensemble runs strictly only store the log items that are requested. I guess, we can just add these to the dict in `ensemble.py` in case the `save_network` switch is set and not execute these lines here if the `save_network` switch is not set.

Thanks :)